android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -395,7 +395,9 @@ public String getInstallReferrer() {
     actMgr.getMemoryInfo(memInfo);
     constants.put("totalMemory", memInfo.totalMem);
     constants.put("deviceType", deviceType.getValue());
-
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
I like the API-gate here for the array version of Build.SUPPORTED_ABIS, but this means that in lower APIs (down to API16, React-Native's lower bound) we will not have an entry in constants which will force a logic branch to any consumers of the API, and they won't get any info. I think on the earlier devices you can make an empty array then use Build.CPU_ABI like so https://stackoverflow.com/a/51775541 but add it as [0] in the array

I intend to use this feature almost immediately in my app, down to ABI16 to fetch arch-specific flavors in rn-update-apk so this is important to me, and I'll also serve as a test for it pretty quickly :-). Thanks!

Reply:
fixed.

Reply:
I tested it on a the emulator with API level 16. Fallback seems to work. I'm hoping it'll work as expected on the device too.